src/main/java/io/stargate/sgv2/jsonapi/api/model/command/VectorSortable.java

```diff
@@ -20,10 +22,15 @@ default Optional<Boolean> includeSortVector() {
    *
    * @return the vector sort expression if it exists.
    */
-  default Optional<SortExpression> vectorSortExpression() {
-    if (sortClause() != null && sortClause().sortExpressions() != null) {
+  default Optional<SortExpression> vectorSortExpression(CommandContext<?> ctx) {
+    return vectorSortExpression(ctx.schemaObject());
+  }
+
+  default Optional<SortExpression> vectorSortExpression(SchemaObject schema) {
```

Comment:
Similar to `SortClause`, 1 caller has full `CommandContext` to pass, another (`TableSimilarityFunction`)  just `TableSchemaObject`.